netcore/Makefile

```diff
@@ -129,48 +129,15 @@ xtest-%: prepare update-corefx-tests
 		-notrait category=outerloop -notrait category=nonosxtests -notrait category=failing \
 		-notrait category=nonnetcoreapptests -notrait category=nonmonotests \
 		-html ../../TestResult-$*.html -nunit ../../TestResult-$*-netcore-xunit.xml \
-		$(shell if [ -a $(CURDIR)/excludes-$*.rsp ]; then grep -v '^#\|^$$' $(CURDIR)/excludes-$*.rsp; fi;) \
+		$(shell if [ -a $(CURDIR)/CoreFX.issues.rsp ]; then grep -v '^#\|^$$' $(CURDIR)/CoreFX.issues.rsp; fi;) \
 		$(FIXTURE) || true
 
 # these tests won't be included in `xtestall`
-EXCLUDED_COREFX_TESTS = \
-	System.Utf8String.Experimental.Tests \
-	System.ComponentModel.Composition.Registration.Tests
-
-# Tests suites which do pass yet
-EXCLUDED_COREFX_TESTS_NOT_PASSING = \
-	$(EXCLUDED_COREFX_TESTS) \
-	System.ComponentModel.Composition.Tests \
-	System.Composition.TypedParts.Tests \
-	System.Console.Tests \
-	System.Data.Common.Tests \
-	System.Data.SqlClient.Tests \
-	System.Diagnostics.StackTrace.Tests \
-	System.Drawing.Common.Tests \
-	System.Numerics.Vectors.Tests \
-	System.Reflection.Context.Tests \
-	System.Reflection.CoreCLR.Tests \
-	System.Reflection.Emit.ILGeneration.Tests \
-	System.Reflection.Emit.Lightweight.Tests \
-	System.Reflection.Metadata.Tests \
-	System.Reflection.MetadataLoadContext.Tests \
-	System.Reflection.TypeExtensions.CoreCLR.Tests \
-	System.Reflection.TypeExtensions.Tests \
-	System.Runtime.Extensions.Tests \
-	System.Runtime.InteropServices.Tests \
-	System.Runtime.Loader.DefaultContext.Tests \
-	System.Runtime.Loader.RefEmitLoadContext.Tests \
-	System.Runtime.Loader.Tests \
-	System.Runtime.Serialization.Formatters.Tests \
-	System.Threading.Overlapped.Tests \
-	System.Xml.Xsl.XslTransformApi.Tests
+EXCLUDED_COREFX_TESTS =
 
 xtestall: update-corefx-tests $(foreach workingtest, $(foreach test, $(wildcard corefx/tests/extracted/*), \
 		$(filter-out $(EXCLUDED_COREFX_TESTS), $(notdir $(test)))), $(addprefix xtest-, $(workingtest)))
 
-xtestpass: update-corefx-tests $(foreach workingtest, $(foreach test, $(wildcard corefx/tests/extracted/*), \
```

Comment:
Are we abandoning `xtestpass` altogether now? The main reason for it was to maintain a list for the CI. I know it was never deployed, but the general idea was to have a known subset with 0 failures. If `xtestall` can fill that role I am fine with it.

Reply:
@filipnavara yeah, I removed it because xtestall now should have 0 failures.